db/migrate/20170518162321_create_assignments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAssignments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :assignments do |t|
+      t.timestamps null: false
+      t.references :outcome_coverage, index: true, foreign_key: true, null: false
+      t.string :name, null: false
+      t.string :problem, null: false
```

Comment:
From the mockup, I think this is supposed to be optional? https://marvelapp.com/8b0f34e/screen/27979251